src/NBitcoin/NetworkOptions.cs

```diff
@@ -31,11 +45,53 @@ public NetworkOptions(uint flags = All)
             this.flags = flags;
         }
 
+        /// <summary>
+        /// Get/Set ProofOfStake flags.
+        /// </summary>
+        public bool IsProofOfStake
+        {
+            get
+            {
+                bool isPOS = (this.flags & POS) != 0;
+                // This sanity check will be removed once the static flags are removed
+                if (isPOS != Block.BlockSignature)
+                {
+                    throw new ArgumentException($"Block.BlockSignature { Block.BlockSignature} mismatches cross-check value: { isPOS }");
```

Comment:
There's a extra space after the curly brace `BlockSignature { Block`

Reply:
Done